enterprise/api/v1/views/enterprise_group.py

```diff
@@ -133,6 +143,35 @@ def create(self, request, *args, **kwargs):
         """
         return super().create(request, *args, **kwargs)
 
+    @action(methods=['patch'], detail=False, permission_classes=[permissions.IsAuthenticated])
+    @permission_required('enterprise.can_access_admin_dashboard', fn=lambda request, group_uuid: group_uuid)
+    def update_learner(self, request, group_uuid):
+        """
+        Endpoint location to Update one, or more, fields for a learner:
+        PATCH api/v1/enterprise-group/<group_uuid>/learners/
+
+        Request Arguments:
+        - ``group_uuid`` (URL location, required): The uuid of the group which learners should be updated
+
+        """
+        group = self.get_queryset().get(uuid=group_uuid)
+        data = self.request.data
+        learner = data.get("learner")
+        try:
+            pecu_in_q = Q(pending_enterprise_customer_user__user_email=learner)
```

Comment:
Why are we constraining this endpoint to only include pending learners?

Reply:
I updated this method more clear in our intent `update_pending_learner_status`. this endpoint only serves the reminder email updates from `enterprise-access`, which we are only sending to pending learners since existing enterprise customers automatically have the `accepted` status